cfme/utils/net.py

```diff
@@ -66,8 +66,9 @@ def net_check(port, addr=None, force=False):
     if port not in _ports[addr] or force:
         # First try DNS resolution
         try:
-            addr = socket.gethostbyname(addr)
-
+            addr_info = socket.getaddrinfo(addr, port)[0]
+            sockaddr = addr_info[4]
+            addr = sockaddr[0]
```

Comment:
This should work for `addr` being hostname, IPv4 and IPv6.
I followed advice by @rhrazdil.